liferay-workspace/tests/blade.samples.test/src/testIntegration/java/com/liferay/blade/samples/test/BladeCLIUtil.java

```diff
@@ -140,8 +140,11 @@ public static String getLatestBladeCLIJar() throws Exception {
 
 	public static String installBundle(File file) throws Exception {
 		String output = execute("sh", "install", file.toURI().toString());
-
-		String bundleID = output.substring(output.length() - 3);
+		
+		String bundleID = output.substring(
+			output.indexOf("bundle id:") + 11,
```

Comment:
@gamerson , FYI, there was change made to the blade CLI. the install output has an extra newline at the end. That was causing my tests to break, so I improved how i got the bundle id.